modules/ROOT/pages/introduction/use-exchange.adoc

```diff
@@ -0,0 +1,192 @@
+= Use Anypont Exchange to Discover Connectors, Templates, and Examples
+
+ifndef::env-site,env-github[]
+include::_attributes.adoc[]
+endif::[]
+:imagesdir: ../../assets/images/
+
+Use Anypoint Exchange to discover integration assets such as connectors, templates, and examples and import them into your app or project. You can browse and search for assets built and curated by MuleSoft or assets available in your own organization's Exchange.
+
+To access assets in MuleSoft's Exchange, do either of the following:
```

Comment:
To avoid making a trademark possessive, I suggest using "MuleSoft" here as a plain adjective:
```suggestion
To access assets in MuleSoft Exchange, do either of the following:
```

Reply:
I guess we refer to "Exchange" only rather than "Exchange instance"? It sounds odd to me to say "your organization's Exchange," but I have had other writers correct me. Would you please let me know your thoughts about this in a separate Slack message?

Reply:
Saying "MuleSoft Exchange" sounds odd to me. As for saying "Exchange" rather than "Exchange instance", I don't usually work with Exchange, but that's how I've seen it in the doc.